trezorlib/stellar.py

```diff
@@ -0,0 +1,52 @@
+import base64
+import struct
+import binascii
+
+def get_index_from_account_number(account):
+    """Returns an integer index or the default of 0"""
+    max_account_index = pow(2, 31) - 1
+
+    if account is None:
+        index = 0
+    else:
+        index = int(account) - 1
+
+    if index < 0 or index > max_account_index:
+        raise ValueError("Error: invalid account number (must be between 1 and " + str(max_account_index) + ")")
+
+    return index
+
+
+def get_public_key_from_bytes(pk_bytes):
```

Comment:
Wouldn't `address_from_public_key` be a better name?

Reply:
Agreed, renamed to `address_from_public_key`